python/ray/state.py

```diff
@@ -385,6 +385,66 @@ def client_table(self):
 
         return _parse_client_table(self.redis_client)
 
+    def _job_table(self, job_id):
+        """Fetch and parse the job table information for a single job ID.
+
+         Args:
+            job_id: A job ID to get information about.
```

Comment:
Should note that it can be a `JobID` or hex.

Reply:
fixed

Reply:
not actually fixed, right?

Reply:
yeah, it got rolled back